tests/benchmark/Makefile

```diff
@@ -41,8 +41,8 @@ LDFLAGS += ${CRYPTO_LDFLAGS} ${LIBS} ${CRYPTO_LIBS} ${BENCHMARK_LDFLAGS} -lm -ld
 
 $(BM_TESTS)::
```

Comment:
Any improvements in benchmarks from the change?

Reply:
pem parsing is now running in 6ns vs 12ns previously